discovery-infra/start_discovery.py

```diff
@@ -391,6 +393,12 @@ def _get_provisioning_cidr6(cidr, ns_index):
     return str(provisioning_cidr)
 
 
+def _get_network_type(ipv4, ipv6):
+    ipv4_only = ipv4 and not ipv6
+    if ipv4_only:
```

Comment:
if not ipv6:

Reply:
what about 'auto-assign'? shouldn't it be the default?

Reply:
no,

Reply:
I'm a bit confused here, but maybe I'm outdated about recent events
Don't we use OVNKubernetes for ipv6?
Because it's kind of reverse in here